ft_connectivityanalysis.m

```diff
@@ -66,6 +67,8 @@
 %                     method 'powcorr' and 'amplcorr'.
 %   cfg.bandwidth   = scalar, needed for 'psi', half-bandwidth of the integration
 %                     across frequencies (in Hz, default is the Rayleigh frequency)
+%                     needed for 'plm', half-bandwidth of the integration window (in Hz)
+%   cfg.fsample     = scalar, needed for 'plm', sampling frequency of the data in [Hz]
```

Comment:
If the input data structure is of type 'raw', the fsample is typically present, or can be easily deduced from the time-axis. I believe that ft_checkdata has an option that does this (and can thus be included in line 399). In other words, there should be no need to define the fsample separately in the cfg.

Reply:
I see now that fsample is not necessarily present in 'raw' data.